src/main/cpp/auton/PrimitiveParams.cpp

```diff
@@ -18,6 +18,7 @@
 #include "auton/PrimitiveParams.h"
 #include "chassis/IChassis.h"
 #include "chassis/ChassisOptionEnums.h"
+#include "DragonVision/DragonCamera.h"
```

Comment:
Why do we need this include added?

Reply:
This include is for the DragonCamera::PIPELINE enum type.  However, we include DragonCamera in DragonVision so it can be removed.

Reply:
Removed